tools/checkpython

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
```

Comment:
checkpatch is documented in README.md. It would be nice to document checkpython too.